CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## DLA-Future 0.7.0
+
+### Changes
+
+- Added (generalized) eigensolver which compute only a part of the eigenspectrum. (#1194)
+- `Norm` is now fully asynchronous (#1221)
+- `stdexec` is now included only in C++ files, allowing builds with CUDA enabled. (#1188)
```

Comment:
Maybe?
```suggestion
- `stdexec` is now included only in C++ files, allowing builds with CUDA and stdexec enabled. (#1188)
```
Just thinking if the current version makes it sound like "builds with CUDA enabled" in general are not allowed. Possibly even "fixing builds with CUDA and stdexec enabled"?